### DISCUSSION TITLE
Help for msg: "OTerror { code: incompatibleLenght,msg:  "expected: 1 but received: 0

### DISCUSSION THREAD

**Discussion by mida-adim** [AppFlowy-IO/AppFlowy#3026] [Technical help wanted]
HI,

... first of all, thank you for this super practical application! I've searched everywhere for an answer to my problem, but unfortunately, I haven't found one. In one of my boards, I mistakenly wrote a message that was too long, so the board no longer displays correctly. Instead, an error message is displayed: msg: 'OTerror { code: incompatibleLength, msg: 'expected: 1 but received: 0'. Is it possible to modify the message that is too long and retrieve the board as it was? If so, how? 

Once again, thank you for this amazing tool!

**Comment by annieappflowy** [comment]
Have you upgraded your application to the latest release?
You can find how to export your data from the current version and import into the latest in the release notes of v0.2.0

Please let me know if you need more help.

**Reply by Xazin** [reply]
Hey @mida-adim, did you try what Annie proposed?

If you still experience this issue, please let us know.

**Reply by mida-adim** [reply]
Hi, thank you for your responses. I updated Appflowy to version 0.27; however, it didn't work, and even worse, I lost all my data. Fortunately, I had copied them, but I can't import them into the new version. When I click on "import" or "change" in the settings, nothing happens; no window opens. Just to clarify, I'm on Ubuntu 22. Thanks again...

**Reply by annieappflowy** [reply]
You need to export your data using v0.1.6 and import your data with v0.2.0
You can find instructions in[ this](https://github.com/AppFlowy-IO/AppFlowy/releases/tag/0.2.0) release notes. Please let me know if you need more help.

**Reply by mida-adim** [reply]
I sincerely apologize for the inconvenience, but I have tried to follow the steps for data import and export, as you can see in my screenshots (especially the video capture). When I click to import or export data, nothing happens!


![Capture d’écran du 2023-08-03 16-20-23](https://github.com/AppFlowy-IO/AppFlowy/assets/139971408/9575cdff-1f52-4515-ac36-ba04b2dd8bb9)


[Capture vidéo du 03-08-2023 16:23:22.webm](https://github.com/AppFlowy-IO/AppFlowy/assets/139971408/ff26ccad-02e6-45e9-83fc-f179ee8c480e)

**Reply by LucasXu0** [reply]
@mida-adim I use Ubuntu 22.04 as well and try installing the AppFlowy v0.2.7 via deb installation. The import panel works fine on my side. How did you install the AppFlowy? zip, deb or flatpak?

https://www.loom.com/share/21b310b3cb1b47428565ae720fd9e411
